CHANGELOG.md

```diff
@@ -77,6 +77,8 @@ Main (unreleased)
   now export receivers immediately, removing the need for dependant components
   to be evaluated twice at process startup. (@rfratto)
 
+- Support Bearer tokens for metric remote writes in the Grafana Operator (@jcreixell, @marctc)
```

Comment:
Is this technically a bugfix since it was already in the CRDs but didn't work?

Reply:
yeah, that's fair, although we noticed other missing fields, expect follow up PRs :)